### PR TITLE
[release] bump 7.9 branch to 7.9.1-SNAPSHOT

### PR DESCRIPTION
**Pull request by ygel**
This PR bump chart version and Docker images used to 7.9.1-SNAPSHOT to test 7.9.1 development version

See [New branching model](https://github.com/elastic/helm-charts/blob/master/BREAKING_CHANGES.md#new-branching-model) for more details.

### CODE REVIEW

**Comment by ygel** [comment]
At least partial success this time:
image

File: elasticsearch/examples/kubernetes-kind/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+This example deploy a 3 nodes Elasticsearch 7.9.1-SNAPSHOT cluster on [Kind][]
 using [custom values][].
 
 Note that this configuration should be used for test only and isn't recommended
 for production.
 
 Note that Kind < 0.7.0 are affected by a [kind issue][] with mount points
 created from PVCs not writable by non-root users. [kubernetes-sigs/kind#1157][]
-fix it in Kind 0.7.0.
+fix it in Kind 0.7.9.1-SNAPSHOT
```
Comment (jmlrt) [inline elasticsearch/examples/kubernetes-kind/README.md:11]: This line shouldn't be changed

File: elasticsearch/examples/upgrade/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # Upgrade
 
 This example will deploy a 3 node Elasticsearch cluster using an old chart version,
-then upgrade it to version 7.9.0.
+then upgrade it to version 7.9.1-SNAPSHOT.
 
 The following upgrades are tested:
 - Upgrade from [7.0.0-alpha1][] version on K8S <1.16
-- Upgrade from [7.4.0][] version on K8S >=1.16 (Elasticsearch chart < 7.4.0 are
+- Upgrade from [7.9.1-SNAPSHOT][] version on K8S >=1.16 (Elasticsearch chart < 7.9.1-SNAPSHOT are
```
Comment (jmlrt) [inline elasticsearch/examples/upgrade/README.md:8]: This line shouldn't be changed

File: elasticsearch/examples/upgrade/README.md [hunk 3]
```diff
@@ -22,6 +22,6 @@ You can also run [goss integration tests][] using `make test`.
 
 
 [7.0.0-alpha1]: https://github.com/elastic/helm-charts/releases/tag/7.0.0-alpha1
-[7.4.0]: https://github.com/elastic/helm-charts/releases/tag/7.4.0
+[7.9.1-SNAPSHOT]: https://github.com/elastic/helm-charts/releases/tag/7.9.1-SNAPSHOT
```
Comment (jmlrt) [inline elasticsearch/examples/upgrade/README.md:25]: This line shouldn't be changed

**Review by jmlrt — CHANGES_REQUESTED** [review]
Some lines that shouldn't be changed. The bumper script need to be reworked to prevent this.

File: elasticsearch/examples/default/test/goss.yaml [hunk 4]
```diff
@@ -15,7 +15,7 @@ http:
     status: 200
     timeout: 2000
     body:
-      - '"number" : "7.9.0"'
+      - '"number" : "7.9.1"'
```
Comment (jmlrt) [inline elasticsearch/examples/default/test/goss.yaml:18]:
```suggestion
      - '"number" : "7.9.1-SNAPSHOT"'
```

related to https://github.com/elastic/helm-charts/blob/ce7f0873a15fb8d80ec2cd3c74eb8547da171f88/elasticsearch/examples/default/test/goss.yaml#L18

File: elasticsearch/examples/oss/test/goss.yaml [hunk 5]
```diff
@@ -11,7 +11,7 @@ http:
     status: 200
     timeout: 2000
     body:
-      - '"number" : "7.9.0"'
+      - '"number" : "7.9.1"'
```
Comment (jmlrt) [inline elasticsearch/examples/oss/test/goss.yaml:14]:
```suggestion
      - '"number" : "7.9.1-SNAPSHOT"'
```

related to https://github.com/elastic/helm-charts/blob/ce7f0873a15fb8d80ec2cd3c74eb8547da171f88/elasticsearch/examples/oss/test/goss.yaml#L14

File: elasticsearch/examples/upgrade/test/goss.yaml [hunk 6]
```diff
@@ -11,7 +11,7 @@ http:
     status: 200
     timeout: 2000
     body:
-      - '"number" : "7.9.0"'
+      - '"number" : "7.9.1"'
```
Comment (jmlrt) [inline elasticsearch/examples/upgrade/test/goss.yaml:14]:
```suggestion
      - '"number" : "7.9.1-SNAPSHOT"'
```

related to https://github.com/elastic/helm-charts/blob/ce7f0873a15fb8d80ec2cd3c74eb8547da171f88/elasticsearch/examples/upgrade/test/goss.yaml#L14

**Review by jmlrt — CHANGES_REQUESTED** [review]
Seems that I forgot to document that Elasticsearch API add `-SNAPSHOT` to the version while the other products don't do it 😭

**Comment by jmlrt** [comment]
@ygel, you'll also need to merge `origin/7.9` on your branch to get https://github.com/elastic/helm-charts/pull/798 which is required to fix CI build.

**Comment by ygel** [comment]
🎉 
```
10:15:04 Finished: SUCCESS
```

**Review by jmlrt — APPROVED** [review]
LGTM⛴

**Comment by jmlrt** [comment]
We need to find a way to prevent or document the manual changes from 3d6ab1da3b150de53823277e48ab0abe7c4676d4

**Comment by ygel** [comment]
please don't delete my 7.9 branch on the fork though :)

**Pull request merged**